hack/functests.sh

```diff
@@ -20,4 +20,4 @@ set -e
 
 source ./cluster/kubevirtci.sh
 
-go test ./tests --kubeconfig $(kubevirtci::kubeconfig) -ginkgo.v
+go test ./tests/... --kubeconfig $(kubevirtci::kubeconfig) -ginkgo.v -test.v -race
```

Comment:
Do we want to run our e2e tests with -race? I would expect it for unit tests, but anything above it sounds like a potential source of flakes. Please keep this PR to the point of "more readable". We can decide to make it -race later.

Reply:
removed `-race`.
Done